components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/resources/web/entitlement/advance-search.jsp

```diff
@@ -88,7 +89,7 @@
             </script>
 
             <script type="text/javascript">
-                showErrorBox("<%=results.getMessage()%>");
+                showErrorBox("<%=Encode.forJavaScriptAttribute(results.getMessage())%>");
```

Comment:
Encode for html and then encode for javascript block. The passed parameter is directly injected to an html